tests/modal.spec.ts

```diff
@@ -0,0 +1,65 @@
+import { test, expect } from '@playwright/test';
+
+test('Renders appropriately according to open attribute', async ({ page }) => {
+  await page.goto('/modal-test.html');
+
+  // No "open" attribute applied
+  await expect(page.getByTestId('modal-default')).toBeHidden()
+
+  // open="false" attribute applied
+  await expect(page.getByTestId('modal-closed')).toBeHidden()
+
+  // open="true" attribute applied
+  await expect(page.getByTestId('modal-open')).toHaveJSProperty('hidden', false)
```

Comment:
using `toBeVisible()` doesn't work here because Playwright's [definition of visible](https://playwright.dev/docs/actionability#visible) marks any element with a `hidden` attribute as _not_ visible, even if the value is `false`

Reply:
Oh weird, it looks like we're not using the `hidden` attribute though, and instead toggling the `invisible` class (and thereby toggling `visibility: hidden;`) to set visibility, and the open modal should not have the CSS `visibility` computed property at all.

Reply:
Maybe it's considering it not visible because the `<v-modal>` element itself has zero size - its children do. Do visibility tests work when instead selecting the first element with a size within the modal instead?

Reply:
you're totally correct, fixed